build-logic/build.gradle.kts

```diff
@@ -4,48 +4,50 @@ import org.jetbrains.kotlin.samWithReceiver.gradle.SamWithReceiverGradleSubplugi
 plugins {
   `embedded-kotlin`
   id("java-gradle-plugin")
+  id("net.mbonnin.golatac").version("0.0.2")
 }
 
 plugins.apply(SamWithReceiverGradleSubplugin::class.java)
 extensions.configure(SamWithReceiverExtension::class.java) {
   annotations(HasImplicitReceiver::class.qualifiedName!!)
 }
 
-group = "com.apollographql.apollo3"
+group = "com.apollographql.apollo3.build"
```

Comment:
This is so that our convention plugins are not taken into account by `checkApolloDuplicates`